traversal/procedure/GraphProcedure.java

```diff
@@ -324,167 +327,167 @@ public ProcedureVertex.Thing setPredicate(ProcedureVertex.Thing thing, Predicate
         public ProcedureEdge.Native.Type.Sub.Forward forwardSub(
                 ProcedureVertex.Type child, ProcedureVertex.Type parent, boolean isTransitive) {
             ProcedureEdge.Native.Type.Sub.Forward edge =
-                    new ProcedureEdge.Native.Type.Sub.Forward(child, parent, -1, isTransitive);
+                    new ProcedureEdge.Native.Type.Sub.Forward(child, parent, isTransitive);
```

Comment:
From here to the end of the file: procedure edges no longer take an `order` parameter.